samples/msal-node-samples/device-code/test/device-code-aad-agc.spec.ts

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import puppeteer from "puppeteer";
+import { Screenshot, createFolder } from "../../../e2eTestUtils/TestUtils";
+import { NodeCacheTestUtils } from "../../../e2eTestUtils/NodeCacheTestUtils";
+import { 
+    approveRemoteConnect,
+    enterCredentials,
+    enterDeviceCode,
+    SCREENSHOT_BASE_FOLDER_NAME,
+    validateCacheLocation
+ } from "../../testUtils";
+import { Configuration, PublicClientApplication } from "../../../../lib/msal-node";
+import { getKeyVaultSecretClient, getCredentials } from "../../../e2eTestUtils/KeyVaultUtils";
+
+// Set test cache name/location
+const TEST_CACHE_LOCATION = `${__dirname}/data/aad-agc.cache.json`;
+
+// Get flow-specific routes from sample application
+const getTokenDeviceCode = require("../index");
+
+// Build cachePlugin
+const cachePlugin = require("../../cachePlugin.js")(TEST_CACHE_LOCATION);
+
+// Load scenario configuration
+const config = require("../config/AAD-AGC.json");
```

Comment:
Is this required for non-AGC runs?  Or are these tests not run in this environment?